app/models/document.rb

```diff
@@ -14,7 +14,15 @@ class Document < ApplicationRecord
     error_sending_to_live
   ].freeze
 
+  REVIEW_STATES = %w[
+    unreviewed
+    submitted_for_review
+    force_published
+    reviewed
+  ].freeze
```

Comment:
Yes, there's a difference. When a document is `submitted_for_review`, the primary action is a publish button (because if you think the document is okay you can publish it). When the document is `published_without_review`, the primary action is the approve button.

I expect that user research might give us some new insights here, but this is how it's current designed.

Reply:
I've added the user facing state to this PR anyway, it probably makes it easier to understand the whole system.